frontend/public/style/_overrides.scss

```diff
@@ -158,6 +154,10 @@ form.pf-c-form {
     flex-direction: column;
   }
 
+  &__main-section {
+    overflow: hidden; // needed for Firefox to enable proper scrolling of content-scrollable
+  }
+
```

Comment:
@mattnolting This was removed recently and somehow snuck back in. It's what's causing the tests to fail.